### DISCUSSION TITLE
Welcome to automuteus Discussions!

### DISCUSSION THREAD

**Discussion by denverquane** [automuteus/automuteus#314] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Teku16** [comment]
Hey there, love the prospect of this! Do you plan to support the Among Us VR when it comes out? Will it be a plug-and-play type of thing or will you have to fully redevelop the game stat poller?

**Reply by kurokobo** [reply]
Among Us VR has in-game voice chat, so in the first place I believe there is no need to use Discord and AutoMuteUs.

> _In-Game Voice Chat: Communicate using voice chat (featuring proximity chat) or use quick text chat_
> https://store.steampowered.com/app/1849900/Among_Us_VR/
